modules/parrableIdSystem.js

```diff
@@ -52,7 +52,7 @@ function fetchId(configParams, consentData, currentStoredId) {
   if (!isValidConfig(configParams)) return;
 
   const data = {
-    eid: currentStoredId || null,
+    eid: currentStoredId ? deserializeParrableId(currentStoredId) : null,
```

Comment:
Since this data object is what is sent to nest, this eid needs to be the actual eid, not our composite cookie object.
I expect this line to read:
```javascript
    eid: currentStoredId ? deserializeParrableId(currentStoredId).eid | null
```